app/src/main/res/xml/preferences_master.xml

```diff
@@ -557,4 +557,17 @@
             android:title="@string/language" />
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/backup_restore">
+        <Preference
+            android:icon="@drawable/ic_save_black_24dp"
+            android:key="@string/pref_key__backup_settings"
+            android:title="@string/backup_settings"
+            android:summary="@string/backup_save_summary" />
+        <Preference
+            android:icon="@drawable/ic_restore_black_24dp"
+            android:key="@string/pref_key__restore_settings"
+            android:title="@string/restore_settings"
+            android:summary="@string/restore_backup_summary" />
+    </PreferenceCategory>
```

Comment:
I have this under the main settings. Not sure if it is better here or under one of the other submenus.

Reply:
First = Important. Second  = Various stuff people barley need